1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,31 @@
+import { fizzbuzz } from "./fizzbuzz";
 
 describe("fizzbuzz", () => {
+  it.each([3, 9, 42])("should return Fizz when given %i", (n) => {
+    expect(fizzbuzz(n)).toEqual("Fizz");
+  });
 
+  it.each([15, 45, 90])("should return Buzz when given %i", (n) => {
+    expect(fizzbuzz(n)).toEqual("FizzBuzz");
+  });
 
+  it.each([5, 10, 20])("should return Buzz when given %i", (n) => {
+    expect(fizzbuzz(n)).toEqual("Buzz");
+  });
+
+  it.each([1, 2, 2224])("should return %i when given %i", (n) => {
+    expect(fizzbuzz(n)).toEqual(n.toString());
+  });
+
+  it.each([undefined, null, {}])("should throw an error when given %i", (n) => {
+    expect(fizzbuzz(n as any)).toEqual(n);
```

Comment:
You can test throwing an error in Jest by wrapping the function in an arrow function:
`expect(() => fizzBuzz("abc")).toThrow("<Error message goes here>")`